docs/en/sql-reference/aggregate-functions/reference/meanztest.md

```diff
@@ -0,0 +1,70 @@
+---
+toc_priority: 303
+toc_title: meanZTest
+---
+
+# meanZTest {#meanztest}
+
+Applies mean z-test to samples from two populations.
+
+**Syntax**
+
+``` sql
+meanZTest(population_variance_x, population_variance_y, confidence_level)(sample_data, sample_index)
+```
+
+Values of both samples are in the `sample_data` column. If `sample_index` equals to 0 then the value in that row belongs to the sample from the first population. Otherwise it belongs to the sample from the second population.
+The null hypothesis is that means of populations are equal. Normal distribution is assumed. Populations may have unequal variance and the variances are known.
```

Comment:
Is is really means that population are equal? I thought that null hypothesis is that only means of two distributions are equal.

Reply:
What is the alternative? Two sided or one sided? E.g. mean1 != mean2 or mean1 >(<) mean2?

Reply:
The alternative is Two sided.

Reply:
I found that the null hypothesis for z-test is the below.
```
Null hypothesis:

  The two sample z test tests the following null hypothesis (H0):
    H0: μ1 = μ2
  Here μ1 is the population mean for group 1, and μ2 is the population mean for group 2.
```
Could you please explain why do you think like that?

Reply:
I somehow misread (I don't know how). I thought that it was written that the null hypothesis is that the two distributions are equal.. Sorry, my fault

Reply:
That's OK. I dont mind. Thank you.